client/src/components/layout/DestinationsIndex.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+import DestinationTile from "./DestinationTile";
+
+const DestinationsIndex = (props) => {
+
+    const [destinations, setDestinations] = useState([])
+
+    const getDestinations = async () => {
+        try {
+            const response = await fetch("/api/v1/destinations")
+            if (!response.ok) {
+                const errorMessage = (`${response.status} ${response.statusText}`)
+                const error = new Error(errorMessage)
+                throw(error)
+            }
+            const body = await response.json()
+            const newDestinations = body.destinations
+            setDestinations(
+                ...destinations,
+                newDestinations
+            )
+        } catch (err) {
+            console.error(`Error in Fetch, ${err.message}`)
+        }
+    }
+
+    useEffect(() => {
+        getDestinations()
+    }, [])
+
+    const destinationTiles = destinations.map((destination) => {
+        return (
+            <p key={destination.id}>
+                <DestinationTile destination={destination}/>
+            </p>
```

Comment:
remove the p tag, an dpass the key to the DestinationTile

If you still want a p tag, it should go IN your component